src/MOI_wrapper.jl

```diff
@@ -28,8 +28,23 @@ end
 DualOptimizer constructor
 """
 function DualOptimizer(dual_optimizer::OT) where {OT <: MOI.ModelLike}
-    return DualOptimizer{OT}(DualProblem(dual_optimizer))
+    dual_problem = DualProblem(dual_optimizer)
```

Comment:
I would call `DualOptimizer{Float64}(dual_optimizer)` instead of copy-pasting the code

Reply:
indeed it makes more sense